pom.xml

```diff
@@ -39,11 +39,11 @@
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
         <maven.gpg.plugin.version>1.6</maven.gpg.plugin.version>
 
-        <spring-data-aerospike.version>3.3.1</spring-data-aerospike.version>
+        <spring-data-aerospike.version>3.3.2-SNAPSHOT</spring-data-aerospike.version>
```

Comment:
Should be replaced with version containing this change:
https://github.com/aerospike-community/spring-data-aerospike/pull/357

Reply:
@Aloren Does "3.3.2-SNAPSHOT" also contains the latest spring-data (parent) version (2.6.x)?, we need to bump spring data parent in Spring-Data-Aerospike as well.

Reply:
changed to 3.4.0 as discussed in https://github.com/aerospike-community/spring-data-aerospike/pull/357
also merged updates with spring-data-* deps.